perl-lib/OESS/lib/OESS/Interface.pm

```diff
@@ -86,24 +86,24 @@ sub to_hash{
 =cut
 sub _fetch_from_db{
     my $self = shift;
-
+    $self->{'logger'}->error("Inside fetch from db");
     if (!defined $self->{'interface_id'}) {
         if (defined $self->{'name'} && defined $self->{'node'}) {
             my $interface_id = OESS::DB::Interface::get_interface(db => $self->{'db'}, interface => $self->{'name'}, node => $self->{'node'});
             if (!defined $interface_id) {
-                $self->{'logger'}->error("Unable to fetch interface $self->{name} on $self->{node} from the db!");
+                $self->{'logger'}->error("sdfsdfUnable to fetch interface $self->{name} on $self->{node} from the db!");
```

Comment:
what is this?

Reply:
I see several places where the logs have had some text appended, could you clean these up.

Reply:
Sure!